src/metrics/instance_definitions_test.go

```diff
@@ -34,3 +34,17 @@ func Test_generateInstanceDefinitions10(t *testing.T) {
 
 	assert.Equal(t, 3, len(queryDefinitions))
 }
+
+func Test_generateInstanceDefinitions170(t *testing.T) {
+	v := semver.MustParse("17.0.0")
+	queryDefinitions := generateInstanceDefinitions(&v)
+
+	assert.Equal(t, 2, len(queryDefinitions))
+}
+
+func Test_generateInstanceDefinitions175(t *testing.T) {
+	v := semver.MustParse("17.5.0")
+	queryDefinitions := generateInstanceDefinitions(&v)
+
+	assert.Equal(t, 2, len(queryDefinitions))
+}
```

Comment:
Since an small refactor was introduced, we can take the chance to test also previous cases `7.0.1`, `9.1.0, `9.1.1`, `9.2.0`, `9.2.3`, ... .  We could use table-driven tests to do so. Something similar to:

```go
test_cases := []struct {
  version string
  expected []*QueryDefinition
} {
  {
    "17.0.0",
    []*QueryDefinition{
			instanceDefinitionBase170,
			instanceDefinition170,
		},
  },
  // additional test cases
}

for _, tc := range test_cases {
  t.Run(tc.version, func(t *testing.T) {
  // generate semver and check that definitions match the expected
  }
}
```